.github/ISSUE_TEMPLATE/feature_request.yml

```diff
@@ -0,0 +1,33 @@
+name: Feature Request
+description: Suggest an idea for the project
+labels:
+  - "enhancement"
+body:
+  - type: markdown
+    attributes:
+      value: "# Please search the issue tracker for existing feature requests before submitting your own."
+  - type: textarea
+    id: enhancement-summary
+    attributes:
+      label: Enhancement Summary
+      description: |
+        - Briefly describe the enhancement.
+        - Explain why you believe the proposed enhancement to be a good idea, and (if applicable) how it helps overcome a limitation of LMMS you are currently facing.
+    validations:
+      required: true
+  - type: textarea
+    id: mockup
+    attributes:
+      label: Implementation Details / Mockup
+      description: |
+        - Explain how you believe this enhancement should be implemented.
+        - If your proposal encompasses changes to the user interface, please include **diagrams** displaying your intent.
```

Comment:
I'm not sure whether "diagrams" needs to be in bold here. No other text besides "Note:" in the other form is bold, and it seems unambiguous enough on its own.